services/web/client/source/class/osparc/dashboard/StudyBrowserButtonItem.js

```diff
@@ -510,10 +511,28 @@ qx.Class.define("osparc.dashboard.StudyBrowserButtonItem", {
       });
     },
 
-    __setLockedBy: function(lockedBy) {
+    __setLockedBy: function(lockedBy, status) {
       const lock = this.getChildControl("lock");
+      let toolTipText = null;
+      switch (status) {
+        case "CLOSING":
+          toolTipText = lockedBy + this.tr(" is closing it...");
+          break;
+        case "CLONING":
+          toolTipText = lockedBy + this.tr(" is cloning it...");
+          break;
+        case "EXPORTING":
+          toolTipText = lockedBy + this.tr(" is exporting it...");
+          break;
+        case "OPENING":
+          toolTipText = lockedBy + this.tr(" is opening it...");
+          break;
+        case "OPENED":
+          toolTipText = lockedBy + this.tr(" is using it.");
+          break;
```

Comment:
if these messages are shown to the user, maybe we should have less simplistic and more friendly wording?

Reply:
this will be handled by @odeimaiz in a separate PR.